DESCRIPTION

```diff
@@ -46,7 +46,8 @@ Suggests:
     repurrrsive (>= 1.0.0),
     rmarkdown,
     readr,
-    testthat (>= 2.1.0)
+    testthat (>= 2.1.0),
+    forcats
```

Comment:
We try to avoid adding dependencies/suggests solely for use in examples. You've done a great job of describing how it _can_ work with `fct_drop()`, but it's rare enough, that the actual example might not be worth it in this case.